packages/legend-application-studio/src/components/editor/edit-panel/GrammarTextEditor.tsx

```diff
@@ -703,6 +705,17 @@ export const GrammarTextEditor = observer(() => {
 
   const { ref, width, height } = useResizeDetector<HTMLDivElement>();
 
+  useEffect(() => {
```

Comment:
let's not do this here, but in `grammarTextEditorState.setWrapText`, simply do

```ts
setWrapText(val: boolean): void {
  this.wrapText = val;
  this.editorStore.applicationStore.storageService.persistSetting(LEGEND_STUDIO_SETTINGS_KEY.EDITOR_WRAP_TEXT, val);
}
```

Reply:
However, as noted in `StorageService.ts`, I'm leaning more and more towards putting this `explicit` persistence to storage in the `toggleWordWrap()` method here instead of the in the setter in state. What do you think?